lib/Service/EncodingService.php

```diff
@@ -77,6 +77,7 @@ private function detectUtfBom(string $string): ?string {
 	 */
 	private function getEncodings(): array {
 		$mbOrder = mb_detect_order() ?: [];
```

Comment:
Seems like this line can go, no?

Reply:
The return value is still needed, unfortunately the return value of mb_detect_order is strange as it is documented to return a bool|array while when calling it without an argument it will always return an array. I've updated the relevant line to require the duplicate variable assignment